wandb/sdk/interface/interface.py

```diff
@@ -75,6 +76,26 @@ def file_enum_to_policy(enum: "pb.FilesItem.PolicyType.V") -> "PolicyName":
     return policy
 
 
+def artifact_policy_to_enum(
+    policy: "ArtifactPolicy",
+) -> "pb.ArtifactManifestEntry.Policy.V":
+    if policy == "mutable":
+        enum = pb.ArtifactManifestEntry.Policy.MUTABLE
+    elif policy == "immutable":
+        enum = pb.ArtifactManifestEntry.Policy.IMMUTABLE
+    return enum
```

Comment:
does this work? common switch idiom in Python:

```suggestion
    return {
        "mutable": pb.ArtifactManifestEntry.Policy.MUTABLE,
        "immutable": pb.ArtifactManifestEntry.Policy.IMMUTABLE,
    }[policy]
```

likewise for the other direction

this has the added benefit of raising a KeyError on an invalid value, whereas the current code just returns None (which isn't documented on the return type)